src/main/java/mate/jdbc/Main.java

```diff
@@ -11,32 +11,32 @@
 
 public class Main {
     private static final Injector injector = Injector.getInstance("mate.jdbc");
+    private static final ManufacturerService manufacturerService
+            = (ManufacturerService) injector.getInstance(ManufacturerService.class);
+    private static final DriverService driverService
+            = (DriverService) injector.getInstance(DriverService.class);
+    private static final CarService carService
+            = (CarService) injector.getInstance(CarService.class);
+    private static final Manufacturer manufacturer = new Manufacturer("BMW", "USA");
+    private static Car car = new Car();
 
     public static void main(String[] args) {
-        ManufacturerService manufacturerService = (ManufacturerService)
-                injector.getInstance(ManufacturerService.class);
-        DriverService driverService = (DriverService)
-                injector.getInstance(DriverService.class);
-        Manufacturer manufacturer = new Manufacturer("BMW", "USA");
         manufacturerService.create(manufacturer);
         Driver firstDriver = new Driver("Alex", "123");
         Driver seconfDriver = new Driver("Pavel", "456");
         driverService.create(firstDriver);
         driverService.create(seconfDriver);
         System.out.println(driverService.getAll());
-
-        Car car = new Car();
         car.setModel("Mercedes");
         car.setManufacturer(manufacturer);
```

Comment:
You can use constructor with parameters instead of default constructor with setters.
Please fix similar cases if needed.